docs/mobile-apps/virtual-usb.md

```diff
@@ -15,164 +15,247 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
   ]}>
 
-<TabItem value="Android">
+  <TabItem value="Android">
 
-```java
-37D274BC3A65A34BB3DA4DDF7B77E341        Motorola Moto Z     ANDROID     7.0     https://app.eu-central-1.saucelabs.com/live/mobile/dataCenters/EU/devices/shared/9299h0c88a7-e2b6-41bc-9509-5-8a5d765490371e2c9a
+  ```java
+  java -jar virtual-usb-client-2.0.0.jar startSession --username john.smith --accessKey ab015c1e-1997-4794-8g52-32f7fe110e03 --deviceName Motorola_Moto_Z_real
+  ```
 
-localhost:7000  online
-```
-</TabItem>
-<TabItem value="iOS">
+  </TabItem>
+  <TabItem value="iOS">
 
-```java
-37D274BC3A65A34BB3DA4DDF7B77E341        iPad 2             Mac         11.00    https://app.eu-central-1.saucelabs.com/live/mobile/dataCenters/EU/devices/shared/9299h0c88a7-e2b6-41bc-9509-5-8a5d765490371e2c9a
-```
+  ```java
+  java -jar virtual-usb-client-2.0.0.jar startSession --username john.smith --accessKey ab015c1e-1997-4794-8g52-32f7fe110e03 --deviceName iPhone_XS
+  ```
 
-</TabItem>
-</Tabs>
+  </TabItem>
+  </Tabs>
 
->**NOTE**: If you're testing on iOS, skip to the last step.
+   or
 
-8. **Android Only**: Copy the port number and use it to connect adb to your session device by running `adb connect` followed by the `<IPAddress>:<portNumber>` from the previous step:
+   **Option 2: Use Sauce Connect Proxy**
+   * Open a new command line terminal window and start a tunnel.
+    ```sh
+    $ bin/sc --username john.smith --accessKey ab015c1e-1997-4794-8g52-32f7fe110e03 https://saucelabs.com/rest/v1/ --tunnel-identifier awesometunnel
+    ```
+   * Open another new terminal window (meaning, you should have three terminal windows open) and add the same variables as Option 1, plus your `--tunnel-identifier` at the end. The vUSB client will use it to retrieve and secure your test data.
+    ```java
+    java -jar virtual-usb-client-2.0.0.jar startSession --username john.smith --accessKey ab015c1e-1997-4794-8g52-32f7fe110e03 --deviceName Motorola_Moto_Z_real --tunnel-identifier awesometunnel
+    ```
 
-```java
-adb connect localhost:7000
-```
+7. If your vUSB test session launch is successful, you'll see a success message containing:
 
-9. **Android Only**: Now, your vUSB should be fully connected. You can use Android Studio (or Google Chrome's Remote Debugging) to debug your app, execute automation based on adb, or any other tool that is adb-compliant. For example, using adb shell, you can start the camera of the connected device:
+  <Tabs
+    defaultValue="Android"
+    values={[
+      {label: 'Android', value: 'Android'},
+      {label: 'iOS', value: 'iOS'},
+    ]}>
 
-```java
-adb shell
-am start -a android.media.action.IMAGE_CAPTURE
-```
+  <TabItem value="Android">
 
-10. **Android Only**: Run your tests. Remember, you can use the `--help` flag to view a full CLI reference guide for vUSB functionality.
+  Your **session ID**, a **port number**, and a **link** that takes you to your device and allows you to watch your test running in real time.
```

Comment:
We might need to mention that the **link** will only work when you are logged in

Reply:
done